### PR TITLE
Add capstone lessons

### PR DESCRIPTION
**Pull request by reenas**

### CODE REVIEW

**Review by avishekjana — APPROVED** [review]
It looks good to me.

File: 11-capstone/01-about [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+As the culmination of your hard work, it's time to showcase your expertise by presenting your capstone project. This project has been a canvas for you to apply the knowledge and skills acquired across the ten modules, allowing you to create a unique and meaningful application.
+
+To recap, you've delved into the following modules and incorporated their concepts into your project:
+
+1. Workflow using Pull-Requests: You've learned how to open a pull request, make changes, submit work for review, and update code based on feedback.
+
+2. JS Bundling & Integration: You've understood the necessity of "bundling" and explored various tools, applying JavaScript bundling concepts to your project.
+
+3. Compile-to-JS Languages: You've grasped the differences between compile-to-JS languages (TypeScript and Babel) and made an informed choice for your project.
```
Comment (mjzac) [inline 11-capstone/01-about:11]:
```suggestion
3. Compile-to-JS Languages: You've grasped the differences between TypeScript and Babel, and made an informed choice for your project.
```

File: 11-capstone/01-about [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+5. CI/CD Pipeline: You've established a CI/CD pipeline to automate the deployment process, ensuring seamless integration of code changes and swift error detection.
+
+6. Application Environments: You've deployed your application in cluster mode using PM2, implemented environment variable configurations for development and production, and applied security measures to safeguard your application.
+
+7. Containerization: You've containerized your application using Docker, configured environment variables within the containers, defined a Docker Compose file for multiple services (including a database service), and set up a CI/CD pipeline to deploy the Dockerized application on a server.
+
+8. Internationalization & Localization: You've implemented internationalization and localization in your web application, catering to diverse user preferences and geographical locations.
+
+9. Error Logging & Debugging: You've followed error logging and debugging practices, enabling dynamic content translation, localized date and time formats, and provided the ability to switch between different locales.
```
Comment (mjzac) [inline 11-capstone/01-about:23]:
```suggestion
9. Error Logging & Debugging: You've followed error logging and debugging practices,  which helps in finding and resolving bugs as early as possible.
```

**Review by mjzac — CHANGES_REQUESTED** [review]
I have corrected the descriptions for few levels

**Review by mjzac — APPROVED** [review]

**Pull request merged**